workflow-templates/im-test-sam-yaml.yml

```diff
@@ -0,0 +1,52 @@
+# Workflow Code: DigitalWerewolf_v1    DO NOT REMOVE
+# Purpose:
+#    Runs the SAM YAML Validation action and exports the
+#    output JSON into the Stewardship Mapping document repository
+#
+# Frequency:
+#    - This workflow should only be used once per repository
+#
+# Projects to use this Template with:
+#    -  Any project that needs to report SAM information for Stewardship Mapping
+#       See https://github.com/im-practices/master-architecture-plan/blob/main/Implementation%20Decisions/adr-015_system_architecture_metadata.md
+#       for more information
+#
+# TODO: Prerequisites:
+#    - Create a sam.yaml file that describes the Bounded Context project information
+
+name: SAM.yaml Validate and Export to Stewardship Mapping
+
+on:
+  # After a PR is merged, the workflow will check for changes to the SAM YAML file in the root directory
+  pull_request:
+    types: [closed]
```

Comment:
This is based on the conversation in our channel but the workflow/action I've written to do this is not going to trigger this because it didn't create PRs, it just committed directly to the default branch.  Should we modify this trigger to be a push: branches: main instead?  Should we also add a workflow_dispatch in case it needs to be re-run because it is not correct?